src/components/CurrentLocationInfo.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import './CurrentLocationInfo.css';
+
+class CurrentLocationInfo extends React.Component {
+  render() {
+    const { location } = this.props;
+
+    return (
+      <div className='currentLocation'>
+        <div className='title'>
+          Current location:
+        </div>
+        <div className='location'>
+          {location}
+        </div>
+      </div>
+    )
+  }
+};
+
+CurrentLocationInfo.propTypes = {
+  location: PropTypes.string
+};
+
+const mapStateToProps = (state) => {
+  return {
+    location: state.location.city
```

Comment:
`reducers` should have method to extract this information. During refactoring this can be broken